thunder/torch/__init__.py

```diff
@@ -1486,7 +1486,8 @@ def selu(a: TensorProxy, /, inplace: bool = False) -> TensorLike:
 
 
 @torchsymbol(torch.nn.functional.silu)
-def silu(a, /):
+def silu(a: TensorLike, inplace: bool = False):
+    utils.check(not inplace, lambda: "silu only supports inplace=False", exception_type=NotImplementedError)
```

Comment:
I might be nice to include "Thunder" somewhere in the error message so people know it's us not them...